### ISSUE TITLE
MobyLinuxVM Won't Start Causing Docker to Not Start - Windows 10 Pro

### ISSUE THREAD

**Issue by hsnodgrass** [docker/for-win#565]
### Expected behavior
Docker starts.
### Actual behavior
Docker does not start due to MobyLinuxVM not starting.
### Information

  - Diagnostic ID - 5EF774A1-A5E1-453B-B05E-C15EB3685B76/2017-03-09_10-58-06
  - Docker Version - 17.03.0-ce-win1 (10296), Stable Channel
  - Windows Version - Windows 10 Professional build 14393 

  - Log:
Version: 17.03.0-ce-win1 (10296)
Channel: stable
Sha1: e5a07a1a69a5123c71e24ca2aa9e6f875fa26d08
Started on: 2017/03/09 10:55:11.371
Resources: C:\Program Files\Docker\Docker\Resources
OS: Windows 10 Pro
Edition: Professional
Id: 1607
Build: 14393
BuildLabName: 14393.693.amd64fre.rs1_release.161220-1747
File: C:\Users\XXXXX\AppData\Local\Docker\log.txt
CommandLine: "C:\Program Files\Docker\Docker\Docker for Windows.exe" 
You can send feedback, including this log file, at https://github.com/docker/for-win/issues
[10:55:11.471][GUI            ][Info   ] Starting...
[10:55:11.707][Tracking       ][Info   ] Crash report and usage statistics are enabled
[10:55:11.710][SegmentApi     ][Info   ] Usage statistic: appLaunched
[10:55:11.809][NamedPipeClient][Info   ] Sending Version()...
[10:55:11.825][NamedPipeClient][Info   ] Received response for Version
[10:55:11.825][SegmentApi     ][Info   ] Usage statistic: heartbeat
[10:55:08.377][BackendServer  ][Info   ] Started
[10:55:11.825][NamedPipeServer][Info   ] Version()
[10:55:11.825][NamedPipeServer][Info   ] Version done in 00:00:00.
[10:55:12.110][SegmentApi     ][Info   ] Usage statistic: eventNotElectedForFeature
[10:55:12.110][SegmentApi     ][Info   ] Usage statistic: stateFeatureDisabled
[10:55:12.173][Updater        ][Info   ] Checking for updates on channel stable...
[10:55:12.173][Notifications  ][Info   ] Docker is starting...
[10:55:12.188][NamedPipeClient][Info   ] Sending Start(Docker.Core.Settings)...
[10:55:12.188][NamedPipeServer][Info   ] Start(Docker.Core.Settings)
[10:55:12.188][PowerMode      ][Info   ] Stop
[10:55:12.188][HyperV         ][Info   ] Stop
[10:55:12.188][PowerShell     ][Info   ] Run script with parameters: -Stop True...
[10:55:12.204][PowerShell     ][Info   ] Creating a Runspace Pool...
[10:55:12.257][Updater        ][Info   ] Local build 10296 is as good as the remote 10296 on channel stable
[10:55:12.326][PowerShell     ][Info   ] Runspace Pool created (Min=1, Max=2)
[10:55:12.389][HyperV         ][Info   ] Script started at 10:55:12.389
[10:55:13.562][HyperV         ][Info   ] Module loaded at 10:55:13.562
[10:55:13.669][HyperV         ][Info   ] VM MobyLinuxVM does not exist
[10:55:13.669][HyperV         ][Debug  ] [stop] took 00:00:01.4807832 to run
[10:55:13.670][OptimizeDisk   ][Info   ] Optimize
[10:55:13.670][PowerShell     ][Info   ] Run script...
[10:55:13.681][Moby           ][Info   ] Stop
[10:55:13.695][HyperVGuids    ][Info   ] Installing GUIDs...
[10:55:13.695][PowerMode      ][Info   ] Start
[10:55:13.696][HyperV         ][Info   ] Create
[10:55:13.696][PowerShell     ][Info   ] Run script with parameters: -Create True -VhdPathOverride  -SwitchSubnetAddress 10.0.75.0 -SwitchSubnetMaskSize 24 -CPUs 2 -Memory 2048 -IsoFile C:\Program Files\Docker\Docker\Resources\mobylinux.iso...
[10:55:13.697][HyperVGuids    ][Info   ] GUIDs installed
[10:55:13.698][Firewall       ][Info   ] Removing all existing rules...
[10:55:13.704][HyperV         ][Info   ] Script started at 10:55:13.704
[10:55:13.713][HyperV         ][Info   ] Module loaded at 10:55:13.713
[10:55:13.737][Firewall       ][Info   ] All existing rules are removed.
[10:55:13.738][Firewall       ][Info   ] Opening ports for C:\Program Files\Docker\Docker\Resources\com.docker.proxy.exe...
[10:55:13.743][Firewall       ][Info   ] Opening ports for SMB...
[10:55:13.750][Firewall       ][Info   ] Ports are opened
[10:55:13.939][HyperV         ][Info   ] Creating Switch: DockerNAT...
[10:55:15.331][HyperV         ][Info   ] Switch created.
[10:55:15.701][HyperV         ][Info   ] Set IP address on switch
[10:55:16.863][HyperV         ][Info   ] Creating VM MobyLinuxVM...
[10:55:17.514][HyperV         ][Info   ] Setting CPUs to 2 and Memory to 2048 MB
[10:55:17.637][HyperV         ][Info   ] Creating dynamic VHD: C:\Users\Public\Documents\Hyper-V\Virtual Hard Disks\MobyLinuxVM.vhdx
[10:55:18.732][HyperV         ][Info   ] Attach VHD C:\Users\Public\Documents\Hyper-V\Virtual Hard Disks\MobyLinuxVM.vhdx
[10:55:19.201][HyperV         ][Info   ] Connect Internal Switch DockerNAT
[10:55:19.339][HyperV         ][Info   ] Attach DVD C:\Program Files\Docker\Docker\Resources\mobylinux.iso
[10:55:19.802][HyperV         ][Info   ] Disabled Guest Service Interface
[10:55:19.818][HyperV         ][Info   ] Enabled Heartbeat
[10:55:19.840][HyperV         ][Info   ] Disabled Key-Value Pair Exchange
[10:55:19.871][HyperV         ][Info   ] Enabled Shutdown
[10:55:19.871][HyperV         ][Info   ] Enabled Time Synchronization
[10:55:19.887][HyperV         ][Info   ] Disabled VSS
[10:55:20.103][HyperV         ][Info   ] VM created.
[10:55:20.103][HyperV         ][Debug  ] [create] took 00:00:06.4072178 to run
[10:55:20.103][PowerShell     ][Info   ] Run script '(Get-VM MobyLinuxVM).Id.Guid'...
[10:55:20.140][DataKit        ][Info   ] Starting C:\Program Files\Docker\Docker\Resources\com.docker.db.exe --url hyperv-connect://b2b0a884-fddb-4ccb-8b78-ca065e5795ba,\\.\pipe\dockerDataBase --color never
[10:55:20.140][DataKit        ][Info   ] Started
[10:55:20.156][Database       ][Info   ] Generating CA Cert Bundle
[10:55:20.172][DataKit        ][Info   ] Starting com.docker.db.exe 0.7.0-34-g6f04213 ...
[10:55:20.172][Database       ][Info   ] CA Cert Bundle Generated
[10:55:20.187][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/config=static
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/address=10.0.75.2
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/netmask=255.255.255.0
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/gateway=10.0.75.1
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/network=hybrid
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/docker/daemon.json={"registry-mirrors":[],"insecure-registries":[]}
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/sysctl.conf=
[10:55:20.187][Database       ][Info   ] Writing com.docker.driver.amd64-linux/native/port-forwarding=true
[10:55:20.213][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:20.213][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:20.531][Database       ][Info   ] Successfully wrote 8 value(s)
[10:55:20.533][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/ssl/certs/ca-certificates.crt=C:\Windows\TEMP\tmp38F7.tmp
[10:55:20.552][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:20.552][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:20.632][Database       ][Info   ] Successfully wrote value
[10:55:20.636][Moby           ][Info   ] Connecting...
[10:55:20.637][Moby           ][Error  ] Failed to read Moby's logs: The operation has timed out.
[10:55:20.638][HyperV         ][Info   ] Start
[10:55:20.638][PowerShell     ][Info   ] Run script with parameters: -Start True...
[10:55:20.640][PowerShell     ][Info   ] Run script '$(Find-NetRoute -RemoteIPAddress 8.8.8.8).InterfaceIndex[0]'...
[10:55:20.640][HyperV         ][Info   ] Script started at 10:55:20.640
[10:55:20.671][HyperV         ][Info   ] Module loaded at 10:55:20.671
[10:55:20.671][HyperV         ][Info   ] Starting VM MobyLinuxVM...
[10:55:21.057][Database       ][Info   ] Writing com.docker.driver.amd64-linux/slirp/dns=nameserver 10.101.1.1
[10:55:21.087][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:21.087][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:21.145][Database       ][Info   ] Successfully wrote 1 value(s)
[10:55:21.160][VpnKit         ][Info   ] Starting C:\Program Files\Docker\Docker\Resources\com.docker.slirp.exe --ethernet hyperv-connect://b2b0a884-fddb-4ccb-8b78-ca065e5795ba --port hyperv-connect://b2b0a884-fddb-4ccb-8b78-ca065e5795ba --db \\.\pipe\dockerDataBase --debug --diagnostics \\.\pipe\dockerVpnKitDiagnostics
[10:55:21.160][VpnKit         ][Info   ] Started
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: Logging to stdout (stdout:true DEBUG:false)
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: Setting handler to ignore all SIGPIPE signals
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: vpnkit version c41f7c8589352c95b14de636c895e8fbd72222e5 with hostnet version   uwt version 0.0.3 hvsock version 0.13.0 
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: starting port forwarding server on port_control_url:hyperv-connect://b2b0a884-fddb-4ccb-8b78-ca065e5795ba vsock_path:
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: connecting to b2b0a884-fddb-4ccb-8b78-ca065e5795ba:0B95756A-9985-48AD-9470-78E060895BE7
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: connecting to b2b0a884-fddb-4ccb-8b78-ca065e5795ba:30D48B34-7D27-4B0B-AAAF-BBBED334DD59
[10:55:21.176][VpnKit         ][Error  ] com.docker.slirp.exe: While watching /etc/resolv.conf: ENOENT
[10:55:21.176][VpnKit         ][Info   ] com.docker.slirp.exe: hosts file has bindings for 
[10:55:35.800][Linux          ][Error  ] Failed to start: Unable to start: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: 'MobyLinuxVM' failed to start. (Virtual machine ID B2B0A884-FDDB-4CCB-8B78-CA065E5795BA)
at Start-MobyLinuxVM, <No file>: line 287
at <ScriptBlock>, <No file>: line 382. Let's retry
[10:55:35.816][PowerShell     ][Info   ] Run script...
[10:55:38.246][HyperV         ][Info   ] Hyper-V is running
[10:55:38.246][PowerMode      ][Info   ] Stop
[10:55:38.246][HyperV         ][Info   ] Stop
[10:55:38.246][PowerShell     ][Info   ] Run script with parameters: -Stop True...
[10:55:38.261][HyperV         ][Info   ] Script started at 10:55:38.246
[10:55:38.261][HyperV         ][Info   ] Module loaded at 10:55:38.261
[10:55:38.293][HyperV         ][Info   ] VM MobyLinuxVM is stopped
[10:55:38.293][HyperV         ][Debug  ] [stop] took 00:00:00.0468787 to run
[10:55:38.293][OptimizeDisk   ][Info   ] Optimize
[10:55:38.293][PowerShell     ][Info   ] Run script...
[10:55:39.563][Moby           ][Info   ] Stop
[10:55:39.578][VpnKit         ][Info   ] Killing existing com.docker.slirp with PID 1908
[10:55:39.594][DataKit        ][Info   ] Killing existing com.docker.db with PID 5976
[10:55:39.594][HyperV         ][Info   ] Destroy
[10:55:39.594][PowerShell     ][Info   ] Run script with parameters: -Destroy True -KeepVolume True...
[10:55:39.615][HyperV         ][Info   ] Script started at 10:55:39.615
[10:55:39.615][HyperV         ][Info   ] Module loaded at 10:55:39.615
[10:55:39.647][HyperV         ][Info   ] VM MobyLinuxVM is stopped
[10:55:39.647][HyperV         ][Info   ] Destroying Switch DockerNAT...
[10:55:40.111][HyperV         ][Info   ] Removing VM MobyLinuxVM...
[10:55:40.148][HyperV         ][Debug  ] [destroy] took 00:00:00.5541732 to run
[10:55:40.148][Firewall       ][Info   ] Closing ports...
[10:55:40.148][Firewall       ][Info   ] Removing all existing rules...
[10:55:40.233][Firewall       ][Info   ] All existing rules are removed.
[10:55:40.233][Firewall       ][Info   ] Ports are closed
[10:55:40.233][HyperVGuids    ][Info   ] Removing GUIDs...
[10:55:40.233][HyperVGuids    ][Info   ] GUIDs removed
[10:55:40.233][HyperV         ][Info   ] Create
[10:55:40.233][PowerShell     ][Info   ] Run script with parameters: -Create True -VhdPathOverride  -SwitchSubnetAddress 10.0.75.0 -SwitchSubnetMaskSize 24 -CPUs 2 -Memory 2048 -IsoFile C:\Program Files\Docker\Docker\Resources\mobylinux.iso...
[10:55:40.248][HyperV         ][Info   ] Script started at 10:55:40.248
[10:55:40.248][HyperV         ][Info   ] Module loaded at 10:55:40.248
[10:55:40.279][HyperV         ][Info   ] Creating Switch: DockerNAT...
[10:55:41.439][HyperV         ][Info   ] Switch created.
[10:55:41.533][HyperV         ][Info   ] Set IP address on switch
[10:55:41.539][HyperV         ][Info   ] Creating VM MobyLinuxVM...
[10:55:41.940][HyperV         ][Info   ] Setting CPUs to 2 and Memory to 2048 MB
[10:55:42.087][HyperV         ][Info   ] Attach VHD C:\Users\Public\Documents\Hyper-V\Virtual Hard Disks\MobyLinuxVM.vhdx
[10:55:42.433][HyperV         ][Info   ] Connect Internal Switch DockerNAT
[10:55:42.558][HyperV         ][Info   ] Attach DVD C:\Program Files\Docker\Docker\Resources\mobylinux.iso
[10:55:42.905][HyperV         ][Info   ] Disabled Guest Service Interface
[10:55:42.921][HyperV         ][Info   ] Enabled Heartbeat
[10:55:42.943][HyperV         ][Info   ] Disabled Key-Value Pair Exchange
[10:55:42.959][HyperV         ][Info   ] Enabled Shutdown
[10:55:42.959][HyperV         ][Info   ] Enabled Time Synchronization
[10:55:42.974][HyperV         ][Info   ] Disabled VSS
[10:55:43.175][HyperV         ][Info   ] VM created.
[10:55:43.175][HyperV         ][Debug  ] [create] took 00:00:02.9424222 to run
[10:55:43.175][PowerMode      ][Info   ] Stop
[10:55:43.175][HyperV         ][Info   ] Stop
[10:55:43.175][PowerShell     ][Info   ] Run script with parameters: -Stop True...
[10:55:43.175][HyperV         ][Info   ] Script started at 10:55:43.175
[10:55:43.191][HyperV         ][Info   ] Module loaded at 10:55:43.191
[10:55:43.206][HyperV         ][Info   ] VM MobyLinuxVM is stopped
[10:55:43.206][HyperV         ][Debug  ] [stop] took 00:00:00.0312652 to run
[10:55:43.206][OptimizeDisk   ][Info   ] Optimize
[10:55:43.206][PowerShell     ][Info   ] Run script...
[10:55:44.524][Moby           ][Info   ] Stop
[10:55:44.524][HyperV         ][Info   ] Create
[10:55:44.524][PowerMode      ][Info   ] Start
[10:55:44.524][HyperVGuids    ][Info   ] Installing GUIDs...
[10:55:44.524][Firewall       ][Info   ] Removing all existing rules...
[10:55:44.524][PowerShell     ][Info   ] Run script with parameters: -Create True -VhdPathOverride  -SwitchSubnetAddress 10.0.75.0 -SwitchSubnetMaskSize 24 -CPUs 2 -Memory 2048 -IsoFile C:\Program Files\Docker\Docker\Resources\mobylinux.iso...
[10:55:44.524][HyperVGuids    ][Info   ] GUIDs installed
[10:55:44.540][HyperV         ][Info   ] Script started at 10:55:44.524
[10:55:44.546][HyperV         ][Info   ] Module loaded at 10:55:44.546
[10:55:44.562][Firewall       ][Info   ] All existing rules are removed.
[10:55:44.562][Firewall       ][Info   ] Opening ports for C:\Program Files\Docker\Docker\Resources\com.docker.proxy.exe...
[10:55:44.562][Firewall       ][Info   ] Opening ports for SMB...
[10:55:44.578][Firewall       ][Info   ] Ports are opened
[10:55:44.662][HyperV         ][Info   ] Using existing Switch: DockerNAT
[10:55:44.709][HyperV         ][Info   ] Using existing Switch IP address
[10:55:44.725][HyperV         ][Info   ] Setting CPUs to 2 and Memory to 2048 MB
[10:55:44.847][HyperV         ][Info   ] Connect Internal Switch DockerNAT
[10:55:44.947][HyperV         ][Info   ] Disabled Guest Service Interface
[10:55:44.963][HyperV         ][Info   ] Enabled Heartbeat
[10:55:44.979][HyperV         ][Info   ] Disabled Key-Value Pair Exchange
[10:55:44.994][HyperV         ][Info   ] Enabled Shutdown
[10:55:44.994][HyperV         ][Info   ] Enabled Time Synchronization
[10:55:45.010][HyperV         ][Info   ] Disabled VSS
[10:55:45.095][HyperV         ][Info   ] VM created.
[10:55:45.095][HyperV         ][Debug  ] [create] took 00:00:00.5703899 to run
[10:55:45.095][PowerShell     ][Info   ] Run script '(Get-VM MobyLinuxVM).Id.Guid'...
[10:55:45.110][DataKit        ][Info   ] Starting C:\Program Files\Docker\Docker\Resources\com.docker.db.exe --url hyperv-connect://f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832,\\.\pipe\dockerDataBase --color never
[10:55:45.110][DataKit        ][Info   ] Started
[10:55:45.110][Database       ][Info   ] Generating CA Cert Bundle
[10:55:45.126][DataKit        ][Info   ] Starting com.docker.db.exe 0.7.0-34-g6f04213 ...
[10:55:45.142][Database       ][Info   ] CA Cert Bundle Generated
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/config=static
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/address=10.0.75.2
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/netmask=255.255.255.0
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/net/gateway=10.0.75.1
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/network=hybrid
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/docker/daemon.json={"registry-mirrors":[],"insecure-registries":[]}
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/sysctl.conf=
[10:55:45.163][Database       ][Info   ] Writing com.docker.driver.amd64-linux/native/port-forwarding=true
[10:55:45.163][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:45.184][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:45.185][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:45.379][Database       ][Info   ] Successfully wrote 8 value(s)
[10:55:45.379][Database       ][Info   ] Writing com.docker.driver.amd64-linux/etc/ssl/certs/ca-certificates.crt=C:\Windows\TEMP\tmp9A71.tmp
[10:55:45.402][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:45.402][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:45.495][Database       ][Info   ] Successfully wrote value
[10:55:45.495][Moby           ][Info   ] Connecting...
[10:55:45.495][Moby           ][Error  ] Failed to read Moby's logs: The operation has timed out.
[10:55:45.495][HyperV         ][Info   ] Start
[10:55:45.495][PowerShell     ][Info   ] Run script '$(Find-NetRoute -RemoteIPAddress 8.8.8.8).InterfaceIndex[0]'...
[10:55:45.495][PowerShell     ][Info   ] Run script with parameters: -Start True...
[10:55:45.495][HyperV         ][Info   ] Script started at 10:55:45.495
[10:55:45.526][Database       ][Info   ] Writing com.docker.driver.amd64-linux/slirp/dns=nameserver 10.101.1.1
[10:55:45.550][DataKit        ][Info   ] com.docker.db.exe: accepted a new connection on \\.\pipe\dockerDataBase
[10:55:45.551][DataKit        ][Info   ] com.docker.db.exe: Using protocol TwoThousand msize 16384
[10:55:45.642][Database       ][Info   ] Successfully wrote 1 value(s)
[10:55:45.648][VpnKit         ][Info   ] Starting C:\Program Files\Docker\Docker\Resources\com.docker.slirp.exe --ethernet hyperv-connect://f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832 --port hyperv-connect://f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832 --db \\.\pipe\dockerDataBase --debug --diagnostics \\.\pipe\dockerVpnKitDiagnostics
[10:55:45.648][VpnKit         ][Info   ] Started
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: Logging to stdout (stdout:true DEBUG:false)
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: Setting handler to ignore all SIGPIPE signals
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: vpnkit version c41f7c8589352c95b14de636c895e8fbd72222e5 with hostnet version   uwt version 0.0.3 hvsock version 0.13.0 
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: starting port forwarding server on port_control_url:hyperv-connect://f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832 vsock_path:
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: connecting to f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832:0B95756A-9985-48AD-9470-78E060895BE7
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: connecting to f6fa20b0-ee08-4f60-8ce5-4f6d5f95b832:30D48B34-7D27-4B0B-AAAF-BBBED334DD59
[10:55:45.679][VpnKit         ][Error  ] com.docker.slirp.exe: While watching /etc/resolv.conf: ENOENT
[10:55:45.679][VpnKit         ][Info   ] com.docker.slirp.exe: hosts file has bindings for 
[10:55:45.695][HyperV         ][Info   ] Module loaded at 10:55:45.695
[10:55:45.710][HyperV         ][Info   ] Starting VM MobyLinuxVM...
[10:56:00.852][PowerMode      ][Info   ] Stop
[10:56:00.852][HyperV         ][Info   ] Stop
[10:56:00.852][PowerShell     ][Info   ] Run script with parameters: -Stop True...
[10:56:00.868][HyperV         ][Info   ] Script started at 10:56:00.868
[10:56:00.868][HyperV         ][Info   ] Module loaded at 10:56:00.868
[10:56:00.899][HyperV         ][Info   ] VM MobyLinuxVM is stopped
[10:56:00.899][HyperV         ][Debug  ] [stop] took 00:00:00.0468524 to run
[10:56:00.899][OptimizeDisk   ][Info   ] Optimize
[10:56:00.899][PowerShell     ][Info   ] Run script...
[10:56:02.222][Moby           ][Info   ] Stop
[10:56:02.222][VpnKit         ][Info   ] Killing existing com.docker.slirp with PID 11136
[10:56:02.238][DataKit        ][Info   ] Killing existing com.docker.db with PID 5360
[10:56:02.238][HyperV         ][Info   ] Destroy
[10:56:02.238][PowerShell     ][Info   ] Run script with parameters: -Destroy True -KeepVolume True...
[10:56:02.285][HyperV         ][Info   ] Script started at 10:56:02.285
[10:56:02.301][HyperV         ][Info   ] Module loaded at 10:56:02.301
[10:56:02.318][HyperV         ][Info   ] VM MobyLinuxVM is stopped
[10:56:02.319][HyperV         ][Info   ] Destroying Switch DockerNAT...
[10:56:02.755][HyperV         ][Info   ] Removing VM MobyLinuxVM...
[10:56:02.786][HyperV         ][Debug  ] [destroy] took 00:00:00.5476992 to run
[10:56:02.786][Firewall       ][Info   ] Closing ports...
[10:56:02.786][Firewall       ][Info   ] Removing all existing rules...
[10:56:02.870][Firewall       ][Info   ] All existing rules are removed.
[10:56:02.870][Firewall       ][Info   ] Ports are closed
[10:56:02.870][HyperVGuids    ][Info   ] Removing GUIDs...
[10:56:02.870][HyperVGuids    ][Info   ] GUIDs removed
[10:56:02.870][NamedPipeServer][Error  ] Unable to execute Start: Unable to start: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: 'MobyLinuxVM' failed to start. (Virtual machine ID F6FA20B0-EE08-4F60-8CE5-4F6D5F95B832)
at Start-MobyLinuxVM, <No file>: line 287
at <ScriptBlock>, <No file>: line 382    at Docker.Backend.ContainerEngine.Linux.DoStart(Settings settings)
   at Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
   at Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass8_0.<Register>b__0(Object[] parameters)
   at Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)
[10:56:02.902][NamedPipeClient][Error  ] Unable to send Start: Unable to start: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: 'MobyLinuxVM' failed to start. (Virtual machine ID F6FA20B0-EE08-4F60-8CE5-4F6D5F95B832)
at Start-MobyLinuxVM, <No file>: line 287
at <ScriptBlock>, <No file>: line 382
[10:56:02.902][Notifications  ][Error  ] Unable to start: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: 'MobyLinuxVM' failed to start. (Virtual machine ID F6FA20B0-EE08-4F60-8CE5-4F6D5F95B832)
at Start-MobyLinuxVM, <No file>: line 287
at <ScriptBlock>, <No file>: line 382
[10:57:57.314][SegmentApi     ][Info   ] Usage statistic: eventElectedForFeature
[10:57:57.314][SegmentApi     ][Info   ] Usage statistic: stateFeatureEnabled
[10:57:58.765][SegmentApi     ][Info   ] Usage statistic: actionMenuFeedback
[10:58:06.277][FeedbackSettings][Info   ] Upload diagnostic
[10:58:06.281][CrashReport    ][Info   ] Preparing package to send with the diagnostics
[10:58:06.282][NamedPipeClient][Info   ] Sending DownloadVpnKitLogs(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\pcap.tar)...
[10:58:06.282][NamedPipeClient][Info   ] Sending DownloadVmLogs(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\mobyLogs.tar)...
[10:58:06.282][NamedPipeClient][Info   ] Sending DownloadWinContainersDebugInfo(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\win_containers)...
[10:58:06.283][NamedPipeClient][Info   ] Sending GetDebugInfo()...
[10:58:06.283][NamedPipeServer][Info   ] DownloadVpnKitLogs(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\pcap.tar)
[10:58:06.283][NamedPipeServer][Info   ] DownloadVmLogs(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\mobyLogs.tar)
[10:58:06.286][NamedPipeClient][Info   ] Received response for DownloadVpnKitLogs
[10:58:06.283][NamedPipeServer][Info   ] DownloadWinContainersDebugInfo(C:\Users\hesto\AppData\Local\Temp\y1xef3f0.jnr\win_containers)
[10:58:06.284][NamedPipeServer][Info   ] GetDebugInfo()
[10:58:06.286][VpnKit         ][Warning] Unable to download logs: The operation has timed out.
[10:58:06.286][NamedPipeServer][Info   ] DownloadVpnKitLogs done in 00:00:00.0024987.
[10:58:06.300][PowerShell     ][Info   ] Run script '(Get-VM MobyLinuxVM).Id.Guid'...
[10:58:06.313][NamedPipeClient][Info   ] Received response for DownloadVmLogs
[10:58:06.313][HyperV         ][Warning] Unable to download logs: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Hyper-V was unable to find a virtual machine with name "MobyLinuxVM".
[10:58:06.313][NamedPipeServer][Info   ] DownloadVmLogs done in 00:00:00.0295049.
[10:58:13.187][NamedPipeClient][Info   ] Received response for DownloadWinContainersDebugInfo
[10:58:13.187][NamedPipeServer][Info   ] DownloadWinContainersDebugInfo done in 00:00:06.9040138.
[10:58:16.420][NamedPipeClient][Info   ] Received response for GetDebugInfo
[10:58:16.417][NamedPipeServer][Info   ] GetDebugInfo done in 00:00:10.1328393.
[10:58:16.855][CrashReport    ][Info   ] Uploading diagnostics...
[10:58:22.222][CrashReport    ][Info   ] Diagnostics successfully uploaded (5EF774A1-A5E1-453B-B05E-C15EB3685B76/2017-03-09_10-58-06)

### Steps to reproduce the behavior

  1. Starting Docker fails.
  2. Creating MobyLinuxVM with `MobyLinux.ps1 -create` succeeds, but starting the VM with `start-vm -name MobyLinuxVM` fails.
  3. Starting MobyLinuxVM in Hyper-V Manager fails.

**Comment by kmwheeler** [comment]
I experienced the same behavior as what you've reported. Mine started from a fresh install of docker - was never able to run in linux mode but I could switch to windows containers and work from there.  

I was able to solve this problem and get the linux mode working by:
1. un-installing docker
2. removing all of the hyper-v features (windows key + r, type 'optionalfeatures', enter then uncheck all hyper-v items
3. rebooting
4. re-enabling all of the hyper-v features
5. installing docker

I don't know, but I speculate, the way that hyper-v was configured by some process initially caused this behavior. Removing/Adding it using the windows optional features menu seemed to correct this experience for me.

Cheers!
